src/usage_metrics/core/zenodo.py

```diff
@@ -145,6 +109,10 @@ def core_zenodo_logs(
             "Public Utility Data Liberation Project (PUDL) Data Release": "pudl",
             "The Public Utility Data Liberation (PUDL) Project": "pudl",
             "Workplace Democracy, Open Data, and Open Source": "csv_conf_presentation",
+            "Vibrant Clean Energy Resource Adequacy Renewable Energy (RARE) Power Dataset": "vcerare",
+            "The Public Utility Data Liberation Project: Providing Open Data for a Clean Energy Transition": "naps2024",
+            "Removing data barriers for decarbonization with the Public Utility Data Liberation Project": "naps2024",
+            "Vibrant Clean Energy Renewable Generation Profiles": "vcerare",
```

Comment:
Not blocking, but I wonder if there's a way that we could tie these slugs to the concept DOI rather than the archive titles, which may not be stable long term. It would be undesirable for our usage stats to break because someone updated the human-readable title in the source metadata.

Reply:
Yes, that's a much better idea. Updated.